taskcluster/ci/browsertime/kind.yml

```diff
@@ -37,7 +37,7 @@ job-defaults:
                     subject: '[{product_name}] Raptor-Browsertime job "{task_name}" failed'
                     to-addresses: [perftest-alerts@mozilla.com]
             default: {}
-    run-on-tasks-for: []
+    run-on-tasks-for: [github-pull-request]
```

Comment:
If I run `TASK_ID=foo taskgraph target-graph --diff HEAD~1 -p taskcluster/test/params/main-repo-pull-request.yml -J` I get a ton of changes that seem unrelated. Namely, we pull in nightly builds into pull request graphs; I think that's unwanted; we can see those tasks run against this PR. I suspect this may be the line responsible.

Reply:
Ah yes, sorry about that! I added this to run the browsertime tests in the PR to see how they would go. I'll re-run the tests again once the patch hits m-c.